backend/src/util/utilities.ts

```diff
@@ -14,3 +14,22 @@ export const decimalSeparatorByLanguage = (a: Decimal, language: string): string
   i18n.setLocale(rememberLocaleToRestore)
   return result
 }
+
+export const fullName = (firstName: string, lastName: string): string => {
+  return [firstName, lastName].filter(Boolean).join(' ')
+}
+
+export const userName = (f?: string, l?: string): string | null => {
```

Comment:
Please use fullnames for variables:
```suggestion
export const userName = (firstName?: string, lastName?: string): string | null => {
```

Reply:
done